packages/aws-cdk-lib/aws-stepfunctions/test/distributed-map.test.ts

```diff
@@ -237,6 +237,76 @@ describe('Distributed Map State', () => {
     });
   }),
 
+  test('State Machine With Distributed Map State and S3ObjectsItemReaderPath', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const bucketNamePath = '$.bucketName';
+    const prefixPath = '$.prefix';
+
+    //WHEN
+    const map = new stepfunctions.DistributedMap(stack, 'Map State', {
+      maxConcurrency: 1,
+      itemReaderPath: new stepfunctions.S3ObjectsItemReaderPath({
+        bucketNamePath,
+        prefixPath,
+        maxItems: 10,
+      }),
+      itemSelector: {
+        foo: 'foo',
+        bar: stepfunctions.JsonPath.stringAt('$.bar'),
+      },
+    });
+    map.itemProcessor(new stepfunctions.Pass(stack, 'Pass State'));
+
+    //THEN
```

Comment:
Why? There is no `policyStatement` parameter with `IItemReaderPath` unlike `IItemReader`. 

By this logic, we will start checking for absence of everything everywhere just because we know that something similar existed somewhere else.